Client/Frontend/Browser/TabManager.swift

```diff
@@ -19,11 +20,11 @@ class TabManager {
     private var _selectedIndex = -1
     var selectedIndex: Int { return _selectedIndex }
     private let defaultNewTabRequest: NSURLRequest
-
+    
```

Comment:
Xcode has a `Automatically trim traiing whitespace` setting in the *Text Editing* settings. Please enable that to get rid of whitespace only changes like these.